### PR TITLE
Functional tests - Add Js doc for catalog pages

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Add Js doc for catalog pages
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no need
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24995)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

File: tests/UI/pages/BO/catalog/attributes/addValue.js [hunk 1]
```diff
@@ -1,7 +1,16 @@
 require('module-alias/register');
 const BOBasePage = require('@pages/BO/BObasePage');
 
+/**
+ * Add value page, contains functions that can be used on the page
+ * @class
+ * @extends BOBasePage
+ */
 class AddValue extends BOBasePage {
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/catalog/attributes/addValue.js:9]: Some Js doc is missing on this page

File: tests/UI/pages/BO/catalog/categories/index.js [hunk 2]
```diff
@@ -10,8 +10,10 @@ class Categories extends BOBasePage {
 
     // Selectors
     this.editHomeCategoryButton = '#main-div .breadcrumb a[href*=\'edit\']';
+
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/catalog/categories/index.js:22]: The class is missing its description and the constructor too

**Review by boubkerbribri — COMMENTED** [review]
Little feedback about some missing docs. Great improvement.

**Review by Progi1984 — DISMISSED** [review]

File: tests/UI/pages/BO/catalog/attributes/addValue.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/catalog/attributes/addValue.js:9]: done thx :+1:

File: tests/UI/pages/BO/catalog/categories/index.js (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/catalog/categories/index.js:22]: Done thx :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]
Thanks @nesrineabdmouleh

**Review by Progi1984 — CHANGES_REQUESTED** [review]
## WAIT FOR 1780 RELEASE

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh & @boubkerbribri

**Pull request merged**